site/content/en/docs/Installation/Creating Cluster/minikube.md

```diff
@@ -30,7 +30,7 @@ The following command starts a local minikube cluster via virtualbox - but this
 replaced by a [vm-driver](https://github.com/kubernetes/minikube#requirements) of your choice.
 
 ```bash
-minikube start --kubernetes-version v1.16.13 --vm-driver virtualbox
+minikube start --kubernetes-version v{{% k8s-version %}}.13 --vm-driver virtualbox
```

Comment:
minikube-k8s-minor-version

Reply:
These are good ideas. I'll make this change :+1:

Reply:
Done! I like it!